HerPortal.UnitTests/ManagementShell/FixAllUserOwnedConsortiaCommandTests.cs

```diff
@@ -188,6 +203,12 @@ public void MigrateAdmins_IfOwnsConsortia_DoesNothing()
         return (user, expectedLasToRemove, expectedConsortiaToAdd);
     }
 
+    private IEnumerable<(string, List<string>)> GetExampleConsortiumCodesWithCustodianCodes()
+    {
+        yield return ("C_0008", new List<string> { "660", "665" });
+        yield return ("C_0010", new List<string> { "840", "835" });
```

Comment:
Suppose it would still be useful to note that these two consortia are expected to have only these two LAs as members.